localhost.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
You may want to rotate this (if used in your environment) as this would be considered a secret/sensitive.